src/Datadog.Sketches/Mappings/LogarithmicMapping.cs

```diff
@@ -15,14 +16,13 @@ namespace Datadog.Sketches.Mappings;
 /// requires the least number of indices to cover a given range of values.
 /// This is done by logarithmically mapping floating-point values to integers.
 /// </summary>
-public class LogarithmicMapping : IIndexMapping
+public class LogarithmicMapping : IIndexMapping, ISerializable
```

Comment:
For a second I thought this was `System.Runtime.Serialization.ISerializable`. Phew!

Reply:
Yeah, I considered for a while to name it IProtobufSerializable or something like that, but I don't want the abstraction to leak just because the good names were already taken by the BCL 😛